book/src/proposals/durable-tx-nonces.md

```diff
@@ -29,64 +38,75 @@ TODO: svgbob this into a flowchart
 Start
 Create Account
   state = Uninitialized
-InitializeInstruction()
-  if !sysvar.recent_blockhashes.is_empty()
-    stored_hash = sysvar.recent_blockhashes[0]
+NonceInstruction
```

Comment:
for this to work in SLP1, we'll need to create initialized durable Nonce accounts for all the stakers...

Reply:
_This_ being Durable Nonces generally? Doesn't really seem to be a way around that without finding a solution that's not account based